op-chain-ops/upgrades/l1.go

```diff
@@ -698,25 +698,49 @@ func SystemConfig(batch *safe.Batch, implementations superchain.ImplementationLi
 
 	if config != nil {
```

Comment:
We should consider just removing this `if config != nil` check because we know this is only going to run for op mainnet and we have the config